specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/sqlServer.json

```diff
@@ -1432,7 +1432,11 @@
         },
         "isStorageSecondaryKeyInUse": {
           "description": "Specifies whether storageAccountAccessKey value is the storage's secondary key.",
-          "type": "boolean"
+          "type": "boolean",
+          "x-ms-mutability": [
```

Comment:
Is it intentional that this field couldn't read?

refs: https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-mutability